unlock-app/src/components/creator/CreatorLock.js

```diff
@@ -84,13 +95,16 @@ export class CreatorLock extends React.Component {
 CreatorLock.propTypes = {
   lock: UnlockPropTypes.lock,
   transaction: UnlockPropTypes.transaction,
+  keys: UnlockPropTypes.keys,
   config: UnlockPropTypes.configuration,
 }
 
 const mapStateToProps = (state, { lock }) => {
   const transaction = state.transactions[lock.transaction]
+  const keys = state.keys
```

Comment:
hum... why not extract the keys from state inside the component which actually needs them?